packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts

```diff
@@ -58,9 +58,6 @@ export const TitleContainer = styled.div`
 
 export const Title = styled(Text)`
   margin-bottom: 0;
-  white-space: nowrap;
-  text-overflow: ellipsis;
-  overflow: hidden;
   max-width: 600px;
```

Comment:
The designer said that there should be no max-width here, so you should remove this line. You'll need to update also the container grid to make this work if I'm not mistaken.

Reply:
Thanks! Fixed